backend/api/v1/endpoints/Research/UploadPaper.py

```
@@ -0,0 +1,67 @@
+        )
+        db.add(new_paper)
+        db.commit()
+        db.refresh(new_paper)
+
+        return {
+            "message": "Paper uploaded successfully",
+            "paper_id": new_paper.id,
+            "file_name": safe_filename,
+        }
+
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)